GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_Pioneer_NEW.cfg

```diff
@@ -0,0 +1,453 @@
+@PART[bluedog_Pioneer_ProbeCore]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.052
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Probe Core
+	@manufacturer = TRW
+	@description = Breakthroughs in materials and manufacturing have allowed engineers to design a super lightweight hexagonal satellite bus for cheap that can take exploration to new heights. Modular in design, this core is designed with slots to accommodate various expansions like control systems, electrical systems, thermal control systems, experiment bays, and more. <#ef7b06> AUTHOR'S NOTE: Add the Pioneer 10/11 Stellar Reference Assembly to your craft to gain additional SAS modes.</color>
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+	%MODULE[ModuleCommand]
+	{
+		%RESOURCE[ElectricCharge]
+		{
+			%rate = 0.1
+			///stated minimum (after turning off some instruments) is 80W, including comms
+		}
+	}
+	!MODULE[ModuleDataTransmitter]
+	{
+	}
+	!MODULE[ModuleReactionWheel]
+	{
+	}

```

Comment:
This is no longer necessary with RO's new reaction wheel config system <https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RO_ReactionWheels.cfg>.  It will default remove anything that wasn't configured.  Or if you have data on the power draw of the reaction wheel system, you can use that tool to set things up.